templates/flutter/lib/src/input_file.dart.twig

```diff
@@ -1,9 +1,21 @@
-import 'package:http/http.dart' show MultipartFile;
-
 class InputFile {
-  final MultipartFile? file;
-  final String? path;
+  late final String? path;
+  late final List<int>? bytes;
   final String? filename;
+  final String? contentType;
+
+  /// Provide a file using path
+  InputFile({required this.path, this.filename, this.contentType}) {
+    bytes = null;
+  }
+
+  InputFile._internal(this.bytes, {this.filename, this.contentType}) {
+    path = null;
+  }
 
-  InputFile({this.file, this.path, this.filename});
+  /// Provide a file using bytes
+  factory InputFile.fromBytes(
```

Comment:
In JS I tried to support multiple more, can we do the same in all languages? https://github.com/appwrite/sdk-generator/pull/459/files#diff-ca7384a18814afcd9ae2c2d771382d64ddbec1ee5a19f5272bc664a96fcbeee0R10-R38